docs/guides/references/migration-guide.mdx

```diff
@@ -6,7 +6,66 @@ title: Migration Guide
 
 This guide details the changes and how to change your code to migrate to Cypress
 version 13.0.
-[See the full changelog for version 13.0](/guides/references/changelog#13-0-0).
+[See the full changelog for version 13.0](https://docs.cypress.io/guides/overview/why-cypress).
+
+### Cypress Cloud Test Replay
+
+[Test Replay](https://docs.cypress.io/guides/overview/why-cypress) is enabled by default in `v13.0` of the Cypress App.
+
+You may need to whitelist `capture.cypress.io` if you work with a strict VPN. See our FAQ section about [VPN subdomain whitelisting](/faq/questions/cloud-faq#Im-working-with-a-restrictive-VPN-Which-subdomains-do-I-have-to-allow-on-my-VPN-for-Cypress-Cloud-to-work-properly).
```

Comment:
```suggestion
You may need to allowlist `capture.cypress.io` if you work with a strict VPN. See our FAQ section about [VPN subdomain allowlisting](/faq/questions/cloud-faq#Im-working-with-a-restrictive-VPN-Which-subdomains-do-I-have-to-allow-on-my-VPN-for-Cypress-Cloud-to-work-properly).
```

See https://www.linkedin.com/pulse/allowlist-blocklist-better-terms-everyone-lets-use-them-rob-black/

Reply:
I very much appreciate the context of the verbiage change, but I hesitate to introduce a term that isn't widely adopted across the industry.

Reply:
See also GitHub Style Guide:

https://docs.github.com/en/contributing/writing-for-github-docs/style-guide#inclusive-language

I can't say how widely adopted this guidance has become.

Reply:
TIL! 🧠